tests/unit/worker/test_submission_worker.py

```diff
@@ -45,6 +81,22 @@ def test_return_file_url_per_environment(self):
         returned_url = return_file_url_per_environment(self.url)
         self.assertEqual(returned_url, "http://testserver/test/url")
 
+    @mock.patch("scripts.workers.submission_worker.load_challenge")
+    def test_load_challenge_and_return_max_submissions(self, mocked_load_challenge):
+        q_params = {"pk": self.challenge.pk}
+        response = load_challenge_and_return_max_submissions(q_params)
+        mocked_load_challenge.assert_called_with(self.challenge)
+        self.assertEqual(response, (self.challenge.max_concurrent_submission_evaluation, self.challenge))
+
+    def test_load_challenge_and_return_max_submissions_when_challenge_does_not_exist(self):
+        non_existing_challenge_pk = self.challenge.pk + 1
+        with mock.patch("scripts.workers.submission_worker.logger.exception") as mock_logger:
+            try:
+                load_challenge_and_return_max_submissions({"pk": non_existing_challenge_pk})
+            except Challenge.DoesNotExist:
+                pass
+            mock_logger.assert_called_with("Challenge with pk {} doesn't exist".format(non_existing_challenge_pk))
```

Comment:
Should this condition be in the except part?

Reply:
But then the test would be successful if the `Challenge.DoesNotExist` is not raised. So I placed it outside the `try...except` block.

Reply:
Will make some updates to add a `with self.assertRaises` instead.